### DISCUSSION TITLE
toggle theme in light/dark mode

### DISCUSSION THREAD

**Discussion by EllaKaye** [giscus/giscus#653] [Ideas]
Hello!

This is a feature request.

For a website where it is easy to toggle between light and dark modes (e.g. those built with [Quarto](https://quarto.org/docs/output-formats/html-themes.html#dark-mode), could it also be possible to toggle between a light and dark theme for giscus (e.g. by default toggling between GitHub light and GitHub Dark themes)?

I'm not sure if this is something that would need implementing in giscus or Quarto or some combination of the two.

Thanks for considering.

**Comment by laymonage** [comment]
This is already possible by sending a message to the iframe. See the [advanced usage guide](https://github.com/giscus/giscus/blob/main/ADVANCED-USAGE.md#parent-to-giscus-message-events) and comments in #336 and #134.

**Reply by EllaKaye** [reply]
Fantastic! Thanks for getting back to me so quickly and sorry I missed the other relevant discussions before posting.

**Reply by laymonage** [reply]
No worries, let me know if you need any help!
